utility/src/main/java/org/oppia/util/parser/ImageLoader.kt

```diff
@@ -0,0 +1,13 @@
+package org.oppia.util.parser
+
+import android.content.Context
+import android.graphics.Bitmap
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.target.SimpleTarget
+import androidx.annotation.DrawableRes
+
+/** Helper that handles loading and caching images from remote URLs.*/
+interface ImageLoader {
+
+  fun load(context: Context, path: String, target: SimpleTarget<Bitmap>)
```

Comment:
This should document what implementations should do, and what each parameter is for, e.g.:

```kotlin
/** Loads an image from the provided URL into the specified target, optionally caching it. */
```

Can we avoid passing in Context? We should be able to inject that into implementations. Also, should ``path`` be ``imageUrl`` or something similar? 'path' feels local, but the class KDoc indicates this is for downloading remote images.

As an aside, it'd be nice if we could abstract away from ``SimpleTarget`` to fully abstract away Glide, but the current implementation is reasonable. It does help make it easier to migrate away from Glide in the future if we would ever want to.

Reply:
Corrected KDoc comment,  change `path` to `imageUrl`. Removed passing of context and injected in implementation.